inc/class-upgrade.php

```diff
@@ -690,6 +691,33 @@ private function upgrade_128() {
 		Yoast_Notification_Center::get()->remove_notification_by_id( 'wpseo-dismiss-page_comments-notice' );
 	}
 
+	/**
+	 * Performs the xx.x upgrade.
+	 */
+	private function upgrade_xxx() {
+		Yoast_Notification_Center::get()->remove_notification_by_id( 'wpseo-dismiss-onpageorg' );
+
+		// Transfers the onpage option value to the ryte option.
+		$ryte_option   = get_option( 'wpseo_ryte' );
+		$onpage_option = get_option( 'wpseo_onpage' );
+		if ( ! $ryte_option && $onpage_option ) {
+			update_option( 'wpseo_ryte', $onpage_option );
+			delete_option( 'wpseo_onpage' );
+		}
+
+		// Changes onpage_indexability to ryte_indexability.
+		$wpseo_option = get_option( 'wpseo' );
+		if ( isset( $wpseo_option['onpage_indexability'] ) && ! isset( $wpseo_option['ryte_indexability'] ) ) {
+			$wpseo_option['ryte_indexability'] = $wpseo_option['onpage_indexability'];
+			unset( $wpseo_option['onpage_indexability'] );
+			update_option( 'wpseo', $wpseo_option );
```

Comment:
Here and in the lines above: really not  my area of expertise but I'm not sure `get_option()` and `update_option()` are the best choice. Shouldn't be methods from the `WPSEO_Options` class be preferred? I'd like to ask feedback from seniors in the team, when they have a chance 🙂

Reply:
I am not sure if that will work. This is because of `WPSEO_Options` will look if an option exists in the fields that are registered. But maybe it is worth a try. Probably we have to register the old one (onpage_indexability) in `WPSEO_Options_Backfill::get_settings` to have it work. 

Otherwise we possibly can do:
```
                $wpseo_option = get_option( 'wpseo' );
		if ( isset( $wpseo_option['onpage_indexability'] ) && ! isset( $wpseo_option['ryte_indexability'] ) ) {
                     WPSEO_Options::set( 'ryte_indexability', $wpseo_option['onpage_indexability'] );   
		}	
```